model/metrics/system-metrics.yaml

```diff
@@ -340,3 +340,26 @@ groups:
     instrument: updowncounter
     unit: "By"
     attributes: []
+
+  - id: metric.system.linux.memory.slab.usage
+    type: metric
+    metric_name: system.linux.memory.slab.usage
+    stability: experimental
+    brief: "Reports the memory used by the Linux kernel for managing caches of frequently used objects by state."
+    note: |
+      The sum over all `system.linux.memory.slab.usage` state values SHOULD equal the total slab memory available on the system, that is `system.linux.memory.slab.limit`.
+    instrument: updowncounter
+    unit: "By"
+    attributes:
+      - ref: system.linux.memory.slab.state
+
+  - id: metric.system.linux.memory.slab.limit
+    type: metric
+    metric_name: system.linux.memory.slab.limit
+    stability: experimental
+    brief: "Total slab memory used in the system."
```

Comment:
Is it a limit or a total used? 

Limit should be a constant

https://github.com/open-telemetry/semantic-conventions/blob/c80cd1fb36f21eae0321ea1e765d7ae9d036b6d6/docs/general/metrics.md?plain=1#L174-L176

Reply:
Good point. Actually, it is not a constant value, it can grow/shrink over time depending on the used resources. From the shared docs, it seems that we should not be using "limit" for non-constant values, and consequently neither "usage".  As defined in the docs, "total" [should not be](https://github.com/open-telemetry/semantic-conventions/blob/c80cd1fb36f21eae0321ea1e765d7ae9d036b6d6/docs/general/metrics.md?plain=1#L154) used neither.

That makes me wonder if we should rollback the "usage" approach and remove the "limit" metric. I think having a "slab" metric with specific *.state attributes might be the most versatile approach:
```
Metric name: system.linux.memory.slab       
Attributes: system.linux.memory.slab.state (recommended attributes: reclaimable and unreclaimable)
```
Wdyt? cc @open-telemetry/semconv-system-approvers

Reply:
In that case I agree that the `.limit` one can be skipped/removed.

From https://github.com/open-telemetry/semantic-conventions/issues/50, metric names might not be allowed to be namespaces that's why it's most probably better to have `.slab.usage` and `.slab.state`?

So we can report:
```
Metric name: system.linux.memory.slab.usage       
Attributes: system.linux.memory.slab.state (recommended attributes: reclaimable and unreclaimable)
```

Reply:
BTW, do you need limit metric at all since it's derivable from the `system.linux.memory.slab.usage` ?

Reply:
@lmolkova I guess we can make it opt-in instead of recommended, but I think it makes sense to have `.limit`. See #409 for prior discussion on this area for `system.memory.limit`, which is also derivable from `system.memory.usage`

Reply:
+1 for opt-in (if we need it). I believe it's a choice to add a deriveable total-like metric or not add it at all. If there is no use-case for it, then I'd rather not add it, but if you believe it's useful, let's just find a different name and make it opt-in.

Reply:
After discussing this topic in the system SIG, we would opt for removing the limit metric as missing a clear use-case. The total used slab memory is not constant (not a limit), but it can be derivative from the usage state sum (reclaimable + unreclaimable). 

Removed in https://github.com/open-telemetry/semantic-conventions/pull/1078/commits/806a9b6d3fe2b55b560b45e26ccbff9ad7942df5